src/client/utils/tracking/tracking.ts

```diff
@@ -101,15 +100,8 @@ export const { TrackAction, IdentifyAction } = setupTrackers<
 )
 
 type TypeOfContractExcludedUnused = Exclude<
-  TypeOfContract,
-  | 'DK_HOME_CONTENT_OWN'
-  | 'DK_HOME_CONTENT_RENT'
-  | 'DK_HOME_CONTENT_STUDENT_OWN'
-  | 'DK_HOME_CONTENT_STUDENT_RENT'
-  | 'DK_ACCIDENT'
-  | 'DK_ACCIDENT_STUDENT'
-  | 'DK_TRAVEL'
-  | 'DK_TRAVEL_STUDENT'
+  TypeOfContract | NoComboTypes | DkBundleTypes,
+  'DK_ACCIDENT' | 'DK_ACCIDENT_STUDENT' | 'DK_TRAVEL' | 'DK_TRAVEL_STUDENT'
```

Comment:
So basically this type includes all the "promise stacks" that you can sign up for? I don't know any other word for it 😅 The excluded contracts can be added in addition to the core home contents product, yes?

Reply:
Yes! This is one of the confusing things with the existing setup. Since `'DK_ACCIDENT' | 'DK_ACCIDENT_STUDENT' | 'DK_TRAVEL' | 'DK_TRAVEL_STUDENT'` are contract types but not sold as standalone products and only in bundles, we don't want to track them.